doc/zephyr.doxyfile.in

```diff
@@ -1978,6 +1978,7 @@ PREDEFINED             = "CONFIG_SYS_CLOCK_EXISTS=y" \
                          "CONFIG_BT_SMP=y" \
                          "CONFIG_USERSPACE=y" \
                          "CONFIG_BT_BREDR=y" \
+                         "CONFIG_FLASH_PAGE_LAYOUT" \
```

Comment:
Please use the same notation as the ones above:
`"CONFIG_FLASH_PAGE_LAYOUT=y"`

Reply:
done